src/ng1/plugins/customscroll/jquery.jscrollpane.js

```diff
@@ -108,6 +108,21 @@ LICENSE-END
 				}
 			};
 
+			// UX Aspects modification to reinitialise when popovers are shown	
+			var timer = null;
+			var observer = new MutationObserver(function() {
```

Comment:
Apologies for the numerous comments, I have been doing a bit of performance investigating around this as it's such a widely used component.

I think we should introduce two new settings, one specifically for the debounce time (i realise i suggested we use the autoReinitialiseDelay, but upon further investigation this is often set to 0 which would make this quite performance heavy). The time should probably be defaulted to something like 500ms.

We should also add one to enable/disabled this feature all together for extra performance where this wont be needed at all.

Apart from that it looks good.